<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1532</width>
    <height>780</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Californian FB</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Collision Detection System v1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(41, 41, 41);
color: rgb(124, 124, 124);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,0,10,2,0,0,0,0,0">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_importVideo">
          <property name="font">
           <font>
            <family>NanumGothic</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{background-color: rgb(75, 75, 75);
selection-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);

border: 1px solid  #ffffff;
border-radius: 3px;
padding-right: 5px; 
padding-left: 5px;
padding-top: 5px;
padding-bottom: 5px;
}

QPushButton:hover
{
	background-color: rgb(125,125,125);
	border: 0px solid  #ffffff;
}</string>
          </property>
          <property name="text">
           <string>영상 불러오기</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_importCamera">
          <property name="font">
           <font>
            <family>NanumGothic</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{background-color: rgb(75, 75, 75);
selection-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);

border: 1px solid  #ffffff;
border-radius: 3px;
padding-right: 5px; 
padding-left: 5px;
padding-top: 5px;
padding-bottom: 5px;
}

QPushButton:hover
{
	background-color: rgb(125,125,125);
	border: 0px solid  #ffffff;
}</string>
          </property>
          <property name="text">
           <string>카메라 불러오기</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_videoInfo">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color : #ffffff</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>130</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>NanumGothic</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color : #ffffff</string>
          </property>
          <property name="title">
           <string>기능설정</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QCheckBox" name="checkBox_collisionDetect">
               <property name="font">
                <font>
                 <family>NanumGothic</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>충돌 감지</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_objectDetect">
               <property name="font">
                <font>
                 <family>NanumGothic</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>객체 탐지</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_mosaicFace">
               <property name="font">
                <font>
                 <family>NanumGothic</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>얼굴 모자이크</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_startVideo">
          <property name="font">
           <font>
            <family>NanumGothic</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{background-color: rgb(75, 75, 75);
selection-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);

border: 1px solid  #ffffff;
border-radius: 3px;
padding-right: 5px; 
padding-left: 5px;
padding-top: 5px;
padding-bottom: 5px;
}

QPushButton:hover
{
	background-color: rgb(125,125,125);
	border: 0px solid  #ffffff;
}</string>
          </property>
          <property name="text">
           <string>재생</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_stopVideo">
          <property name="font">
           <font>
            <family>NanumGothic</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{background-color: rgb(75, 75, 75);
selection-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);

border: 1px solid  #ffffff;
border-radius: 3px;
padding-right: 5px; 
padding-left: 5px;
padding-top: 5px;
padding-bottom: 5px;
}

QPushButton:hover
{
	background-color: rgb(125,125,125);
	border: 0px solid  #ffffff;
}</string>
          </property>
          <property name="text">
           <string>정지</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_restartVideo">
          <property name="font">
           <font>
            <family>NanumGothic</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{background-color: rgb(75, 75, 75);
selection-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);

border: 1px solid  #ffffff;
border-radius: 3px;
padding-right: 5px; 
padding-left: 5px;
padding-top: 5px;
padding-bottom: 5px;
}

QPushButton:hover
{
	background-color: rgb(125,125,125);
	border: 0px solid  #ffffff;
}</string>
          </property>
          <property name="text">
           <string>처음부터 다시 시작</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_mainLogger">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>[System] 영상을 입력하세요.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_videoWindow_2">
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="margin">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_videoLayout">
        <item>
         <widget class="QLabel" name="label_videoWindow">
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="margin">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_radarWindow">
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="margin">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1532</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
